openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java

```diff
@@ -126,6 +133,18 @@ private String getWebhookTypePrefixCondition(String tableName, String typePrefix
         : String.format("%s.webhookType LIKE '%s%%'", tableName, typePrefix);
   }
 
+  private String getPipelineTypePrefixCondition(String tableName, String pipelineType) {
+    pipelineType = escape(pipelineType);
+    if (DatasourceConfig.getInstance().getDriverClass().startsWith("com.mysql.cj.jdbc")) {
```

Comment:
We are hiding some logic here, can we move this into a method of `DatasourceConfig` called `isMySQLConnector()` or something like this?

Reply:
Thank you for your suggestion. I have updated the code.